junit5-engine/src/main/java/org/junit/gen5/engine/junit5/execution/ParameterContextImpl.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.gen5.engine.junit5.execution;
+
+import java.lang.reflect.Parameter;
+import java.util.Optional;
+
+import org.junit.gen5.api.extension.ParameterContext;
+import org.junit.gen5.commons.util.Preconditions;
+import org.junit.gen5.commons.util.ToStringBuilder;
+
+class ParameterContextImpl implements ParameterContext {
```

Comment:
We don't use the "Impl" suffix anywhere in the JUnit 5 code base.

So please change the name to `DefaultParameterContext`.